scripts/001-binutils-PPU.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/sh -e
+#
 # binutils-PPU.sh by Naomi Peori (naomi@peori.ca)
+# Modified by luizfernandonb (luizfernando.nb@outlook.com)
 
 BINUTILS="binutils-2.22"
 
 if [ ! -d ${BINUTILS} ]; then
 
   ## Download the source code.
-  if [ ! -f ${BINUTILS}.tar.bz2 ]; then wget --continue https://ftp.gnu.org/gnu/binutils/${BINUTILS}.tar.bz2; fi
+  if [ ! -f ${BINUTILS}.tar.gz ]; then wget --continue https://ftp.gnu.org/gnu/binutils/${BINUTILS}.tar.gz; fi
```

Comment:
Why?  This just makes the download 34% larger.

Reply:
Didn't get this change either. Just for the speed improvement of decompression? No need to change this here. :+1:

Reply:
The compilation of the toolchain is already very slow as a whole, the `pigz` (tool that only decompresses .tar.gz) speeds up the decompression a little more, it's not much, but it helps, but if you want I can remove the difference in time it wasn't that big between the `pigz` and the `tar` it wasn't that big